tests/test_pipelines_conversational.py

```diff
@@ -276,6 +209,102 @@ def test_integration_torch_conversation_truncated_history(self):
         self.assertEqual(result.past_user_inputs[1], "Is it an action movie?")
         self.assertEqual(result.generated_responses[1], "It's a comedy.")
 
+    @require_torch
+    @slow
+    def test_integration_torch_conversation_dialogpt_input_ids(self):
```

Comment:
This test worked before this PR, this checks that we did not change behavior for DialoGPT.